Source/bmalloc/libpas/src/libpas/pas_probabilistic_guard_malloc_allocator.h

```diff
@@ -77,6 +77,11 @@ struct pas_pgm_storage {
      */
     bool right_align;
 
+    /*
```

Comment:
Nit: This can be a single line comment. The others I did just because they were multiple lines.

Reply:
Fixed